### PR TITLE
Added ConvertToSingleElementArray transformer

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/24081

## Description
Added a transformer to convert a single string to a single element array with that string.

## Minimum version of Demisto
5.0.0

## Does it break backward compatibility?
No

### CODE REVIEW

**Comment by idovandijk** [comment]
@michalgold need a reviewer

File: Packs/Legacy/Scripts/ConvertToArray/ConvertToArray.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+args = demisto.args()
+value = args.get("value")
+if value and isinstance(value, list):
+    demisto.results(value)
+    elif value:
+        demisto.results([value])
+    else:
+        demisto.results([])
```
Comment (ronykoz) [inline Packs/Legacy/Scripts/ConvertToArray/ConvertToArray.py:8]: shift + tab

**Comment by idovandijk** [comment]
@kirbles19 Only need 1 line + changelog:
Changelog: https://github.com/demisto/content/blob/8252362d31a304feaf5b9bc85ea8b0aa3731f478/Packs/Legacy/Scripts/ConvertToSingleElementArray/CHANGELOG.md

Line 10:
https://github.com/demisto/content/blob/8252362d31a304feaf5b9bc85ea8b0aa3731f478/Packs/Legacy/Scripts/ConvertToSingleElementArray/ConvertToSingleElementArray.yml

**Comment by kirbles19** [comment]
@idovandijk Done.

**Pull request merged**